webflow-hacks/misc-hacks/28/28.ts

```diff
@@ -0,0 +1,15 @@
+// when DOM is ready
+document.addEventListener('DOMContentLoaded', function () {
+  // get the element
+  const INPUT_SELECTOR = '[fs-hacks-element="name-input"]';
+  const TEXT_SELECTOR = '[fs-hacks-element="name-text"]';
+  const textElement = document.querySelector<HTMLDivElement>(TEXT_SELECTOR);
+  const inputElement = document.querySelector<HTMLInputElement>(INPUT_SELECTOR);
+  // early return if no element found
+  if (!textElement || !inputElement) return;
+  // on every input field key stroke
+  inputElement.addEventListener('keyup', function () {
```

Comment:
Any reason why you're using `keyup` instead of `input`? I'm curious

Reply:
Never thought of `input`

Reply:
but wouldn't it be better when the user does copy paste

Reply:
changing this to input...